internal/adapters/events_forwarder/noop_forwarder/noop_forwarder.go

```diff
@@ -24,27 +24,80 @@ package noop_forwarder
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/topfreegames/maestro/internal/core/entities/events"
-	"github.com/topfreegames/maestro/internal/core/entities/forwarder"
-
+	entities "github.com/topfreegames/maestro/internal/core/entities/forwarder"
 	"github.com/topfreegames/maestro/internal/core/entities/game_room"
+	"github.com/topfreegames/maestro/internal/core/ports"
+	"github.com/topfreegames/maestro/internal/core/ports/errors"
+
+	pb "github.com/topfreegames/protos/maestro/grpc/generated"
 )
 
-type noopForwarder struct{}
+type noopForwarder struct {
+	forwarderGrpc ports.ForwarderGrpc
+	GrpcClient    pb.GRPCForwarderClient
+}
 
-func NewNoopForwarder() *noopForwarder {
-	return &noopForwarder{}
+func NewNoopForwarder(forwarderGrpc ports.ForwarderGrpc) *noopForwarder {
+	return &noopForwarder{
+		forwarderGrpc: forwarderGrpc,
+	}
 }
 
 // ForwardRoomEvent forwards room events. It receives the room event attributes and forwarder configuration.
-func (f *noopForwarder) ForwardRoomEvent(ctx context.Context, eventAttributes events.RoomEventAttributes, forwarder forwarder.Forwarder) error {
-	return nil
+func (f *noopForwarder) ForwardRoomEvent(ctx context.Context, eventAttributes events.RoomEventAttributes, forwarder entities.Forwarder) error {
+	if eventAttributes.EventType == events.Arbitrary {
+		event := pb.RoomEvent{
+			Room: &pb.Room{
+				Game:     eventAttributes.Game,
+				RoomId:   eventAttributes.RoomId,
+				Host:     eventAttributes.Host,
+				Port:     eventAttributes.Port,
+				Metadata: *fromMapInterfaceToMapString(eventAttributes.Attributes),
+			},
+			EventType: entities.FromForwardTypeToString(forwarder.ForwardType),
+			Metadata:  *fromMapInterfaceToMapString(forwarder.Options.Metadata),
```

Comment:
@lyracampos this metadata should be a merge between `forwarder.Options.Metadata` and `eventAttributes.Other`, you can check the logic in v9 (plugins/grpc/forwarder.go:239)